creator/Documents/CommandBlocks.md

```diff
@@ -6,14 +6,14 @@ ms.prod: gaming
 ---
 # Getting Started with Command Blocks
 
-Command blocks are a specialized type of block within a Minecraft: Bedrock Edition world that allows the use of console commands in game. Command blocks are able to extend and build completely new experiences for players to experience.
+Command blocks are a specialized type of block within a Minecraft: Bedrock Edition world that allows the use of console commands in-game. The same commands that players can type from chat can be run automatically by command blocks using redstone power.
```

Comment:
I changed this because the first sentence is pretty vague and doesn't really give you an idea of what command blocks actually do.

Reply:
I think we might want both of these sentences - but we probably want to retain the notion that we tell Creators that the same commands you run in the console can be run in a Command Block - AND - that's really the heart of a lot of power in Bedrock add-ons (because the latter might not be completely obvious.)